src/select/stories/constants.js

```diff
@@ -65,7 +66,27 @@ const stateOptions = [
   { value: 'WY', label: 'Wyoming' },
 ];
 
+const generateOptions = (count :number) :Object[] => {
```

Comment:
Why not just use like `genRandomString()` from `MockUtils`?

Reply:
Or... use the imported random function above?

Reply:
🤦🏽‍♀️🤦🏽‍♀️